tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergSource.java

```diff
@@ -90,6 +90,9 @@
   public static volatile boolean receivedInlineOnSuccess;
   public static volatile boolean receivedInlineOnError;
 
+  public static volatile PersonalInfo receivedInlineOnSuccessData;
```

Comment:
where's the assertion that consumes these?

Reply:
just added it for conssitency with the error callback